api/src/main/java/org/openmrs/ConceptProposal.java

```diff
@@ -9,276 +9,208 @@
  */
 package org.openmrs;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
 import java.util.Date;
-
 import org.hibernate.envers.Audited;
-import org.openmrs.util.OpenmrsConstants;
 
 /**
  * A ConceptProposal is a temporary holder for concept that should be in the system. When defining
  * an observation, a user can "propose" a new concept if one isn't found already. The proposal is a
  * simple text entry that will be reviewed later. When a proposal is (edited and) accepted, the
  * encounter that prompted this proposal is updated with a new observation pointing at the new (or
- * edited) concept.
  */
+
+@Entity
+@Table(name = "concept_proposal")
 @Audited
-public class ConceptProposal extends BaseOpenmrsObject {
-	
-	public static final long serialVersionUID = 57344L;
-	
-	// Fields
-	
+public class ConceptProposal extends BaseOpenmrsObject implements java.io.Serializable {
```

Comment:
`BaseOpenmrsObject` already implements `Serializable` so the declaration here is redundant. I don't understand why you _deleted_ the `serialVersionUID`, though. That should be defined, albeit maybe changed if this changes the structure (which it shouldn't).